cmd/commands/cli/command.go

```diff
@@ -102,6 +102,7 @@ type cliApp struct {
 const redColor = "\033[31m%s\033[0m"
 const identityDefaultPassphrase = ""
 const statusConnected = "Connected"
+const youMustAgree = "You must agree with provider and consumer terms of use in order to use this command"
```

Comment:
`
var ErrAgreementNeeded = errors.New("You must agree with provider and consumer terms of use in order to use this command")
`
and return the error where appropriate?